content/docs/setup/kubernetes/platform-setup/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Docker For Desktop
+description: Instructions to setup Docker For Desktop for use with Istio.
+weight: 15
+skip_seealso: true
+keywords: [platform-setup,kubernetes,docker-for-desktop]
+---
+
+If you are running istio under docker for desktop's built-in kubernetes, you may need to increase docker's memory limit
```

Comment:
If you want to run Istio under ...

Reply:
Fixed - also just noticed the new subdirectory got truncated from the path, so moved the file